protocol/x/clob/keeper/liquidations.go

```
@@ -271,6 +271,9 @@ func (k Keeper) PlacePerpetualLiquidation(
 			ctx,
 			liquidationOrder,
 		)
+	if err != nil {
```

IIRC we were previously ignoring this error and re-assigning it below